x-pack/platform/plugins/shared/embeddable_alerts_table/public/create_alerts_table_action.test.ts

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Embeddable factories and creation actions are usually untested for other panels, but since there's a bit of authorization logic I added a small test